### PR TITLE
should not use should

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/samson 

# Risks:
 - none

### CODE REVIEW

File: test/models/buddy_check_test.rb [hunk 1]
```diff
@@ -98,7 +98,7 @@
     before do
       job_execution.stubs(:execute!)
       JobExecution.expects(:start_job).returns(job_execution)
-      job_execution.stubs(:setup!).returns(true)
+      job_execution.stubs(:before!).returns(true)
```
Comment (pschambacher) [inline test/models/buddy_check_test.rb]: Oops you did it again!

Comment (grosser) [inline test/models/buddy_check_test.rb]: too many rebases :D

**Comment by grosser** [comment]
all good now!

**Comment by pschambacher** [comment]
:+1:

**Pull request merged**